.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,27 @@
-## Purpose of this PR
+## Purpose of this pull request
 
-<!-- REQUIRED Describe the goal and the type of changes this PR covers. -->
+<!-- REQUIRED Describe the goal and the type of changes this pull request covers. -->
 
-This PR ...
+This pull request ...
 
-## Affected URLs
+## Affected devdocs pages
 
-<!-- REQUIRED List the URLs you are changing. Not needed for large numbers of files. -->
+<!-- REQUIRED List the affected pages on devdocs.magento.com (URLs). Not needed for large numbers of files. -->
 
 - ...
 - ...
 
-## Links to source code
+## Links to Magento source code
 
-<!--  REMOVE THIS SECTION IF NOT USED. If this PR references a file in a Magento codebase repository, add it here. -->
+<!--  OPTIONAL - REMOVE THIS SECTION IF NOT USED. If this pull request references a file in a Magento codebase repository, add it here. -->
```

Comment:
```suggestion
<!--  OPTIONAL - REMOVE THIS SECTION IF NOT USED. If this PR references a file in a Magento codebase repository, add it here. -->
```

Reply:
Dima is trying to get rid of the acronyms with this PR.